packages/universal-hooks-testing-library/package.json

```
@@ -15,13 +20,17 @@
   "license": "MIT",
   "devDependencies": {
     "@jest/types": "^26.6.2",
+    "@saasquatch/haunted-hooks-testing-library": "0.1.0",
```

Make sure to update these:

```suggestion
    "@saasquatch/haunted-hooks-testing-library": "1.0.0",
```

---

Done in 342692b